contrib/gpg.rc

```diff
@@ -29,58 +35,72 @@
+#
+set pgp_clearsign_command="gpg %?p?--pinentry-mode loopback --passphrase-fd 0? --no-verbose --batch --quiet --output - --armor --textmode %?a?--local-user %a? --clearsign %f"
 
-# create a pgp/mime encrypted attachment
+# Create an encrypted attachment (note that some users include the --always-trust option here)
+#
 # set pgp_encrypt_only_command="/usr/lib/neomutt/pgpewrap gpg-2comp -v --batch --output - --encrypt --textmode --armor --always-trust -- -r %r -- %f"
-set pgp_encrypt_only_command="/usr/lib/neomutt/pgpewrap gpg --batch --quiet --no-verbose --output - --encrypt --textmode --armor --always-trust -- -r %r -- %f"
+#
+set pgp_encrypt_only_command="/usr/lib/neomutt/pgpewrap gpg --batch --quiet --no-verbose --output - --textmode --armor -- --recipient %r -- --encrypt %f"
 
-# create a pgp/mime encrypted and signed attachment
+# Create an encrypted and signed attachment (note that some users include the --always-trust option here)
+# 
 # set pgp_encrypt_sign_command="/usr/lib/neomutt/pgpewrap gpg-2comp %?p?--passphrase-fd 0? -v --batch --output - --encrypt --sign %?a?-u %a? --armor --always-trust -- -r %r -- %f"
-set pgp_encrypt_sign_command="/usr/lib/neomutt/pgpewrap gpg %?p?--passphrase-fd 0? --batch --quiet --no-verbose --textmode --output - --encrypt --sign %?a?-u %a? --armor --always-trust -- -r %r -- %f"
+#
+set pgp_encrypt_sign_command="/usr/lib/neomutt/pgpewrap gpg %?p?--pinentry-mode loopback --passphrase-fd 0? --batch --quiet --no-verbose --textmode --output - %?a?--local-user %a? --armor -- --recipient %r -- --sign --encrypt %f"
 
-# import a key into the public key ring
+# Import a key into the public key ring
+#
 set pgp_import_command="gpg --no-verbose --import %f"
 
-# export a key from the public key ring
-set pgp_export_command="gpg --no-verbose --export --armor %r"
+# Export a key from the public key ring
+#
+set pgp_export_command="gpg --no-verbose --armor --export %r"
 
-# verify a key
+# Verify a key
+#
 set pgp_verify_key_command="gpg --verbose --batch --fingerprint --check-sigs %r"
 
-# read in the public key ring
-set pgp_list_pubring_command="gpg --no-verbose --batch --quiet --with-colons --with-fingerprint --with-fingerprint --list-keys %r"
+# Read in the public key ring
+#
+set pgp_list_pubring_command="gpg --no-verbose --batch --quiet --with-colons --with-fingerprint --with-fingerprint --list-keys %r" 
 
-# read in the secret key ring
-set pgp_list_secring_command="gpg --no-verbose --batch --quiet --with-colons --with-fingerprint --with-fingerprint --list-secret-keys %r"
+# Read in the secret key ring
+#
+set pgp_list_secring_command="gpg --no-verbose --batch --quiet --with-colons --with-fingerprint --with-fingerprint --list-secret-keys %r" 
 
-# fetch keys
+# Fetch keys
 # set pgp_getkeys_command="pkspxycwrap %r"
 
 # pattern for good signature - may need to be adapted to locale!
-
-# set pgp_good_sign="^gpgv?: Good signature from "
-
-# OK, here's a version which uses gnupg's message catalog:
```

Comment:
This command was probably useful.

Reply:
I was merging a couple versions, and didn't notice the `v?` difference from this one:
`# set pgp_good_sign="^gpg: Good signature from"`

Indeed the `v?` is good to have.

Reply:
I meant, this comment: `-# OK, here's a version which uses gnupg's message catalog:`